runtime/stdlib/hashalgorithm.go

```diff
@@ -63,9 +83,10 @@ func hashAlgorithmHashFunction(hashAlgoValue interpreter.MemberAccessibleValue)
 
 			locationRange := invocation.LocationRange
 
-			return inter.Config.HashHandler(
+			return hash(
```

Comment:
One of the main contributions of the PR: the stdlib functions do not require the interpreter to provide support anymore. This will make the stdlib usable with other execution mechanisms (e.g. compiler/VM), and make it easier to contribute new standard library types and values.